lib/mongo/db.js

```diff
@@ -0,0 +1,19 @@
+import { MongoClient } from "mongodb";
+
+const URI = process.env.MONGO_URI;
```

Comment:
Since the .env file is not available in the repo, would be awesome if you can create a file called .env.sample with just keys, then it would be easier for me (or other developers) to find out which keys should be available on the env to run the project